coopersystem-fsd/pcc-framework
# Added new content block for Gutenberg

## Description

- Adds new content block in Gutenberg to display created projects

## Steps to test

1. Go to the page editing screen
2. Add a new block to the page content, search for "Projects"

**Expected behavior: The projects created must appear in the body of the edited page
